pkg/test/utils_test.go

```diff
@@ -222,3 +199,72 @@ func find(s string, r []model.FormSubmission) bool {
 
 	return found
 }
+
+// set the environment variables for a test database
+// in the tests we are creating a unique database for testing
+// and removing it after running all the tests
+func setTestDatabase() {
+
+	// save back the MONGODB_URL env variable that it may be used for production
+	// this is sketchy and should be consider in the refactoring. We have the db package and web package getting MONGODB_URL everywhere
+	envmongodburl = os.Getenv("MONGODB_URL")
+
+	// use TEST_MONGODB_URL to create the connection to a new test database
+	mongodburl := os.Getenv("TEST_MONGODB_URL")
```

Comment:
Yes. I agree that this part is quite confusing. I was thinking on getting a TEST_MONGODB_URL in case you wanted to setup other mongodb server for testing (and not only change the name of the db). I got a bit scared of removing useful DBs.